### PR TITLE
Radioactive pulse event gets neutron bursts

### PR DESCRIPTION
**Pull request by RSG250**
[Feature]
## About the PR 
This PR adds a new type of Burst to the Radiation event making use of a type of radiation thats unused

The Neutron Burst
this burst would appear at a lower probability than the normal burst due to how damaging it can bee
this pr also changes 

adjusts regular radiation burst so be longer before dealing damage 
tunes up Neutron Radiation so it 15 seconds of it wont put you in crit from full health as well as add various things needed to prevent irradiation of certain things
adds unique ui thingy displaying your neutron poisoned, its blue
potassium iodide changed so it can reduce the counter of this radiation type like it does normal radiation
makes it so health scanners actually pick up neutron radiation poisoning level

## Why's this needed? 

I over all think the burst event is rather dull with just green spheres, this would add more overall danger by adding blueish circles that may appear at times

## Changelog

```
(u)RSG250:
(*)Green Radioactive Circle event gets a shy friend, Blue Radioactive circle, who has the more deadly NEUTRON RADIATION
```

### CODE REVIEW

**Comment by RSG250** [comment]
Honestly i was debating if i should have left it so Rad protection gear didnt protect against it or not

File: code/modules/events/radiation.dm [hunk 1]
```diff
@@ -19,9 +19,11 @@
 		for (var/pulses = pulse_amt, pulses > 0, pulses--)
 			pulseloc = pick(wormholeturfs)
 			pulse_lifespan = rand(min_pulse_lifespan,max_pulse_lifespan)
-			new /obj/anomaly/radioactive_burst(pulseloc,lifespan = pulse_lifespan)
+			pick(prob(90); new /obj/anomaly/radioactive_burst(pulseloc,lifespan = pulse_lifespan), prob(10); new /obj/anomaly/neutron_burst(pulseloc,lifespan = pulse_lifespan))
```
Comment (ZeWaka) [inline code/modules/events/radiation.dm:22]: that's not how probabilities work

File: code/modules/status_system/statusSystem.dm [hunk 2]
```diff
@@ -689,25 +690,65 @@ var/global/list/statusGroupLimits = list("Food"=4)
 
 				if(2)
 					damage_tox = (2 * prot)
-					damage_brute = (2 * prot)
+					damage_brute = (1 * prot)
```
Comment (ZeWaka) [inline code/modules/status_system/statusSystem.dm:693]: please remove this dumb copied code every time in the switch statement, this is just tox = stage*prot, brute = (stage/2)*prot

**Review by ZeWaka — CHANGES_REQUESTED** [review]

File: code/modules/events/radiation.dm (hunk 1 above)
Comment (RSG250) [reply code/modules/events/radiation.dm:22]: I realize this now, looking at the miraclium proc for reference

File: code/modules/events/radiation.dm [hunk 3]
```diff
@@ -19,9 +19,11 @@
 		for (var/pulses = pulse_amt, pulses > 0, pulses--)
 			pulseloc = pick(wormholeturfs)
 			pulse_lifespan = rand(min_pulse_lifespan,max_pulse_lifespan)
-			new /obj/anomaly/radioactive_burst(pulseloc,lifespan = pulse_lifespan)
+			pick(prob(100); new /obj/anomaly/radioactive_burst(pulseloc,lifespan = pulse_lifespan), prob(50); new /obj/anomaly/neutron_burst(pulseloc,lifespan = pulse_lifespan))
```
Comment (ZeWaka) [inline code/modules/events/radiation.dm:22]: prob100 is useless

**Review by ZeWaka — APPROVED** [review]

File: code/modules/events/radiation.dm (hunk 3 above)
Comment (RSG250) [reply code/modules/events/radiation.dm:22]: tell that to the miracle matter proc lol

**Comment by RSG250** [comment]
🤔 maybe i should shorten the id of Neutron radiation from "neutron_radiation" to "n_radiation"

**Pull request merged**